web/src/settings_components.ts

```diff
@@ -830,6 +830,16 @@ export function save_discard_widget_status_handler(
     }
 }
 
+export function hide_stream_privacy_warning_compose_banner(): void {
+    const $stream_permissions_warning_banner = $(
+        "#stream_permission_settings .stream-permissions-warning-banner",
+    );
+
+    if ($stream_permissions_warning_banner.length > 0) {
+        $stream_permissions_warning_banner.empty();
+    }
+}
```

Comment:
Ideally this should have been in one of the stream specific files, but I guess that would generate an import cycle most probably. Maybe we can just avoid having this function and have the code directly in discard function since its not reused anywhere else currently.